### DISCUSSION TITLE
(converted from an issue) nodemon postinstallation

### DISCUSSION THREAD

**Discussion by thefiddler** [naugtur/can-i-ignore-scripts#9] [General]
```
[ check? ] 'nodemon' needs reviewing
[
  {
    name: 'nodemon',
    version: '2.0.13',
    scripts: { postinstall: 'node bin/postinstall || exit 0' },
    path: 'node_modules/nodemon/'
  }
]
```

This is safe to ignore, `reason: funding`:
https://github.com/remy/nodemon/blob/main/bin/postinstall.js#L22-L25

**Comment by naugtur** [comment]
Thanks for sharing. This would look much better as a pull request!

**Comment by naugtur** [comment]
I'll hide issues on the repo now. 

Don't get me wrong, I appreciate the contribution. 
It's just quicker for both you and me to create a PR to data.json
